include/tins/pdu_cacher.h

```diff
@@ -53,7 +53,7 @@ namespace Tins {
  * etc.
  */
 template <typename T>
-class PDUCacher : public PDU {
+class TINS_API PDUCacher : public PDU {
```

Comment:
Probably not - I just used a "class.*public PDU" regexp...

On 17 February 2016 at 18:00, Matias Fontanini <notifications@github.com>
wrote:

> In include/tins/pdu_cacher.h
> <https://github.com/mfontanini/libtins/pull/129#discussion_r53195386>:
>
> > @@ -53,7 +53,7 @@ namespace Tins {
> >   * etc.
> >   */
> >  template <typename T>
> > -class PDUCacher : public PDU {
> > +class TINS_API PDUCacher : public PDU {
>
> Is this needed for template classes? From the nature of templates, I would
> assume they shouldn't be imported by users of the library, as the full
> definition is always available.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mfontanini/libtins/pull/129/files#r53195386>.
>



-- 
Regards
Einar Jón
+43 650 291 3472